SampleFuzzingJobs/InPlaceFuzzing/inplacefuzzing/inplacefuzzing/inplacefuzzing.cpp

```diff
@@ -0,0 +1,110 @@
+#include <iostream>
```

Comment:
Add top-level comment explaining what this sample is for. (situation where seeds have references to other files).